main.js

```diff
@@ -1,9 +1,49 @@
 /* eslint-disable no-unused-vars */
-var $enterPangram = document.querySelector('#enterPangram')
+
 var $pangram = document.querySelector('#pangram')
 
-$pangram.addEventListener('keydown', copyText)
+var $input = document.querySelector('.input')
+
+var typeThis = 'Jaded zombies acted quaintly but kept driving their oxen forward.'
+
+var letters = typeThis.split('')
+
+document.addEventListener('DOMContentLoaded', createPrompt(letters))
 
-function copyText() {
+var $firstChild = $pangram.children[0]
+//make first array item active
```

Comment:
You can remove comments. Try to rely on good names instead. Comments are only warranted for _extremely_ long/confusing algorithms.